database/migrations/2021_11_29_151352_announcement.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class Announcement extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('announcements', function(Blueprint $table) {
+            $table->id();
+            $table->text('title');
+            $table->text('message');
+            $table->string('status');
```

Comment:
Status should be of type enum with the array of possible values
https://laravel.com/docs/8.x/migrations#column-method-enum